README.md

```diff
@@ -78,25 +84,30 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+
+wat?
 
 ```
 
 ```js
-// Your answer...
+seriously, wat?
+
+just go ahead and mark this one incorrect.
 ```
```

Comment:
Question was asking to define a route for the 4 restful actions related to CRUD:

```js
app.get("/", function(request, response){
  response.send("INDEX");
});
app.post("/", function(request, response){
  response.send("CREATE");
});
app.put("/:id", function(request, response){
  response.send("UPDATE");
});
app.delete("/:id", function(request, response){
  response.send("DELETE");
});
```